modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -575,6 +575,12 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         static Ptr<ResizeNearestNeighborLayer> create(const LayerParams& params);
     };
 
+    class CV_EXPORTS ResizeBilinearLayer : public Layer
```

Comment:
@dmonterom, what do you think about renaming `ResizeNearestNeighborLayer` to `ResizeLayer` and use interpolation strategy as an additional flag? Please add a test similar to nearest neighbor layer

https://github.com/opencv/opencv/blob/5b868ccd829975da5372bf330994553e176aee09/modules/dnn/test/test_tf_importer.cpp#L325-L328

https://github.com/opencv/opencv_extra/blob/b467369489529d8eb2e847c13b117ec4e253a638/testdata/dnn/tensorflow/generate_tf_models.py#L258-L262

Reply:
Hi @dkurt, thats a good idea, I will change it as soon as possible. I will also add the new test soon.

Reply:
@dmonterom, thank you! It also can help us to resolve an issue https://github.com/opencv/opencv/issues/10985 with `Interp` layer from some of Caffe's forks smoother.

Reply:
Hey @dkurt, I made the changes (I also tested the two layers and everything seems ok). If you agree with them i will replace the resize nearest neighbor test with a new one.

Reply:
@dmonterom please add `*_net`, `*_in` and `*_out` files generated by script as a PR to opencv_extra with modified version of the script.